src/api/sso/src/slack-bolt-client.js

```diff
@@ -0,0 +1,52 @@
+const { WebClient, LogLevel } = require('@slack/web-api');
+
+const SLACK_TOKEN = 'your-slack-app-token';
+const CHANNEL_ID = 'channel-or-user-memberId';
+
+const constructMessageBlock = (displayName, githubAvatarUrl) => {
+  const messageBlock = [
+    {
+      type: 'header',
+      text: {
+        type: 'plain_text',
+        emoji: true,
+        text: 'New Member Notification :tada:',
+      },
+    },
+    {
+      type: 'divider',
+    },
+    {
+      type: 'context',
+      elements: [
+        {
+          type: 'image',
+          image_url: `${githubAvatarUrl}`,
+          alt_text: 'github avatar',
+        },
+        {
+          type: 'mrkdwn',
+          text: `*${displayName}* has joined the Telescope community. Be the first one to welcome them`,
+        },
+      ],
+    },
+  ];
+  return messageBlock;
+};
+const publishSignUpMessage = async (displayName, githubAvatarUrl) => {
+  if (!displayName && !githubAvatarUrl) throw new Error('message payload required');
```

Comment:
I think `githubAvatarUrl` will be removed in the future, favouring [github_username](https://github.com/Seneca-CDOT/telescope/blob/dc899cc3dd06849f9679cc90be1eb05e8c10cb5a/src/db/prisma/schema.prisma#L78).